test/jdk/javax/swing/JViewport/bug4546474.java

```diff
@@ -28,46 +28,26 @@
  * @run main bug4546474
  */
 
-import java.awt.Dimension;
-import java.awt.Robot;
-
 import javax.swing.JPanel;
 import javax.swing.JScrollBar;
 import javax.swing.JScrollPane;
-import javax.swing.SwingUtilities;
 
 public class bug4546474 {
     static JScrollPane scrollpane;
     static JScrollBar sbar;
```

Comment:
These two static vars can be local - you don't need to re-test, after that, just make sure it compiles.

Reply:
It does compile with the changes. Thanks for the quick advice